tests/unit/views/pacbio/PacbioRunIndex.spec.js

```diff
@@ -1,15 +1,41 @@
 import PacbioRunIndex from '@/views/pacbio/PacbioRunIndex'
 import Response from '@/api/Response'
 import { mount, store, Data, flushPromises, nextTick } from '@support/testHelper'
+import { createTestingPinia } from '@pinia/testing'
+import { usePacbioRunsStore } from '../../../../src/stores/pacbioRuns'
```

Comment:
You can just use 

`import { usePacbioRunsStore } from '@/src/stores/pacbioRuns'. `

Same in a few other places.